backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/invoice/invoicePaymentAllocation.feature

```diff
@@ -0,0 +1,784 @@
+@from:cucumber
+@paymentAlloc
+Feature: invoice payment allocation
+
+  Background:
+
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+
+    And metasfresh contains M_PricingSystems
```

Comment:
This stepdef AFAIS actually creates a PS with the name defaultPricingSystem... I might be missing something but i was surprised this doesn't create a collision right away..
so anyways, I think the name should be more unique.
(sames about the PL that you create further down)